docs/responsiveness/issue-management.md

```diff
@@ -0,0 +1,54 @@
+# Issue Management Guide
+
+This document outlines some of the key workflows and policies that dotnet/runtime maintainers use to triage issues and manage their backlogs. The purpose of this guide is to:
+
+1. Provide reference documentation to area owners.
+2. Improve transparency on how we handle issues with the wider community.
+3. Set expectations regarding automated issue management policies.
+
+## Issue Triage
+
+Issue triage is the initial phase of the issue management process in which feature area owners
+
+1. Classify the issue: is it a feature request? bug report? user question? is the issue unclear?
+2. Determine viability of the issue: is it likely that we would support the feature request in principle? does the bug report reproduce? is the reported issue by-design behavior? is the issue a duplicate?
+3. Estimate required effort: does the feature design need further refinement? how much work is needed to get the implementation done?
+4. Assign priority and milestones.
+
+An issue is defined as "triaged" if it either gets closed _or_ gets assigned to a milestone. Issue triage is typically a multi-step process that is performed collaboratively by the feature area owners, issue author and supported by our bot infrastructure. While most issues are typically triaged within a day or two of being reported, in some cases it might take longer depending on a variety of circumstances.
+
+### Triage Workflow
+
+1. The issue labeling bot assigns the `untriaged` label and infers an area label for the new issue.
+2. Area owners get notified and begin triage of the issue. 
+3. Area owners complete triage with the following potential outcomes:
+   * Issue is accepted and assigned to a milestone; MsftBot removes the `untriaged` label; relevant labels further classifying the issue are applied by the area owner.
+   * Issue is closed as either duplicate, wontfix, breaking change, answered question, etc.
+   * Issue is unclear or missing information: area owner assigns the `needs more info` label; MsftBot removes the `untriaged` label and adds a post citing this guide. Possible outcomes:
+     * Author responds within 14 days: MsftBot reinstates the `untriaged` label. GOTO step 2.
+     * No activity is recorded: issue closed automatically by MsftBot.
+
```

Comment:
should this say something about how triage decisions are made? what is considered when we decide whether keep an issue or not? (eg., we would take a PR for it)

Reply:
that's actually potentially more interesting than the process details, and likely quite opaque (perhaps even within the team). what is a bug bar? do we have one (implicitly?) 
also, late in the cycle, what is the bug bar? 
what considerations play into our decisions (eg risk, alignment to our strategy, community input, team resources, code complexity..)

how do we use milestones? 
how do we use assignment? (eg., we consciously do not assign most bugs out -- we used to, but it achieved little and sucked the air out of the repo)
what other labels might triage apply eg., up for grabs.

Reply:
> how triage decisions are made

@eiriktsarpalis, I have `Criteria to determine milestone` and `Closing an Issue` sections that are related to triage decisions.

Reply:
> should this say something about how triage decisions are made? what is considered when we decide whether keep an issue or not? (eg., we would take a PR for it)

> that's actually potentially more interesting than the process details, and likely quite opaque (perhaps even within the team). should this say something about how triage decisions are made? 
> what is a bug bar? do we have one (implicitly?)
> how do we use milestones?
> how do we use assignment?
> what other labels might triage apply eg., up for grabs.

My intention with this draft is to emphasize what issue state transitions are possible rather than why or how they are being made. As you mention the latter can vary depending on area, timing, business priorities, etc. From a responsiveness perspective there is value in documenting just the workflows since it provides a rough set of guardrails to maintainers, lets us identify opportunities to introduce automated policies and makes it easier to identify issues where maintainer action is required. 

The linked triage documents from aspnetcore and VSCode are similar in that regard, as they generally don't expand a lot on why many of these decisions are being made. Perhaps then calling it an "Issue Management Guide" is misleading, this could most likely be an "Issue Management Workflows" chapter within a larger guide.